test/TestImportPlaybook.py

```diff
@@ -0,0 +1,20 @@
+"""Test ability to import playbooks."""
+from ansiblelint.runner import Runner
+
+
+def test_task_hook_import_playbook(default_rules_collection):
+    """Assures import_playbook includes are recognized."""
+    playbook_path = (
+        'test/playbook-import/playbook_parent.yml'.
+        format_map(locals())
```

Comment:
what is the purpose of this since there are no variable placeholders in the path string?

Reply:
Thanks. Fixed it already, leftover from previous version.